source/loader/ze_loader.cpp

```diff
@@ -267,37 +267,43 @@ namespace loader
         auto discoveredDrivers = discoverEnabledDrivers();
         std::string loadLibraryErrorValue;
 
-        std::string home_dir;
+        auto log_directory = getenv_string("ZEL_LOADER_LOG_DIR");
+        if (log_directory.empty()) {
+            std::string home_dir;
 #ifdef _WIN32
-        home_dir = getenv_string("USERPROFILE");
-        if (home_dir == ""){
-            auto home_drive = getenv_string("HOMEDRIVE");
-            auto home_path = getenv_string("HOMEPATH");
-            if ((home_drive != "") && (home_path != "")) {
-                home_dir = home_drive + home_path;
-            } else {
-                home_dir = ".";
+            home_dir = getenv_string("USERPROFILE");
+            if (home_dir == ""){
+                auto home_drive = getenv_string("HOMEDRIVE");
+                auto home_path = getenv_string("HOMEPATH");
+                if ((home_drive != "") && (home_path != "")) {
+                    home_dir = home_drive + home_path;
+                } else {
+                    home_dir = ".";
+                }
             }
-        }
+            log_directory = home_dir + "\\" + LOADER_LOG_FILE_DIRECTORY;
 #else
-        home_dir = getenv_string("HOME");
+            home_dir = getenv_string("HOME");
 
-        if (home_dir == "") {
-            auto pwdir = getpwuid(getuid())->pw_dir;
-            home_dir = (pwdir == NULL) ? "." : std::string(pwdir);
-        }
+            if (home_dir == "") {
```

Comment:
This will never be true because lines 275-281 ensure it's set to something.

Reply:
lines 275-281 are guarded by #ifdef _WIN32, they'll only aply on windows.

Reply:
Ah, didn't see that.